public/css/widget/donut-container.less

```diff
@@ -1,6 +1,8 @@
 .donut-container {
   .card();
 
+  margin-bottom: 1em;
```

Comment:
Please add this to the same rule that also applies the right margin.

Reply:
✅